x-pack/plugins/fleet/server/services/epm/packages/get.ts

```diff
@@ -158,7 +158,9 @@ export async function getPackageInfo({
     ? await Registry.fetchInfo(pkgName, resolvedPkgVersion).catch(() => undefined)
     : undefined;
 
-  if (packageInfo) {
+  // We need to get input only packages from source to get all fields
+  // see https://github.com/elastic/package-registry/issues/864
```

Comment:
Thanks for calling this out. 

FWIW https://github.com/elastic/kibana/issues/115032 is prioritized for Sprint 1 of our 8.6 iteration, so we can hopefully leverage some of the work we've done here in the near future.